server/controllers/meeting.js

```diff
@@ -0,0 +1,45 @@
+const Meeting = require("../models/Meeting");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /meeting
+// @desc Create a meeting
+// @access Private
+exports.createMeeting = asyncHandler(async (req, res, next) => {
+  try {
+    const meeting = await Meeting.create(req.body);
+
+    res.status(201).json({
+      success: {
+        meeting: {
+          username: meeting.userId,
+          duration: meeting.duration,
+        },
+      },
+    });
+  } catch {
+    res.status(400);
+    throw new Error("Invalid meeting data");
+  }
```

Comment:
Express-async-handler will catch thrown errors meaning we don't need this try/catch here 👍🏻